modules/troubleshooting-disabling-autoreboot-mco.adoc

```diff
@@ -3,9 +3,20 @@
 // * support/troubleshooting/troubleshooting-operator-issues.adoc
 
 [id="troubleshooting-disabling-autoreboot-mco_{context}"]
-= Disabling Machine Config Operator from automatically rebooting
+= Disabling Machine Config Operator from automatically rebooting	
 
-When configuration changes are made by the Machine Config Operator, {op-system-first} must reboot for the changes to take effect. Whether the configuration change is automatic, such as when a `kube-apiserver-to-kubelet-signer` CA is rotated, or manual, such as when a registry or SSH key is updated, an {op-system} node reboots automatically unless it is paused.
+When configuration changes are made by the Machine Config Operator, {op-system-first} must reboot for the changes to take effect. Whether the configuration change is automatic, such as when a `kube-apiserver-to-kubelet-signer` CA is rotated, or manual, an {op-system} node reboots automatically unless it is paused.
```

Comment:
Suggestions since this content was already in place :) 

s/Machine Config Operator/Machine Config Operator (MCO)
s/CA/certificate authority (CA)